Tests/Admin/AdminTest.php

```diff
@@ -2065,6 +2129,114 @@ public function testGetDataSourceIterator()
         $admin->getDataSourceIterator();
     }
 
+    /**
+     * @expectedException \RuntimeException
+     * @expectedExceptionMessage Circular reference detected! The child admin `sonata.post.admin.post` is already in the parent tree of the `sonata.post.admin.comment` admin.
```

Comment:
Please use https://github.com/sonata-project/SonataAdminBundle/blob/93faa541a3e569aeb301ec57f30893e5a255f9fc/Tests/Helpers/PHPUnit_Framework_TestCase.php#L26 for this